kyuubi-server/pom.xml

```diff
@@ -202,6 +202,16 @@
             </exclusions>
         </dependency>
 
+        <dependency>
+            <groupId>io.swagger.core.v3</groupId>
+            <artifactId>swagger-jaxrs2</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>com.github.swagger-akka-http</groupId>
+            <artifactId>swagger-scala-module_${scala.binary.version}</artifactId>
```

Comment:
That means we need introduce Akka for such feature?

Reply:
No, this groupId named `com.github.swagger-akka-http` is only because it's under organization `swagger-akka-http`. from its [build.sbt](https://github.com/swagger-akka-http/swagger-scala-module/blob/develop/build.sbt#L47-L53) we found it depends on the following libraries.

```
libraryDependencies ++= Seq(
  "org.slf4j" % "slf4j-api" % "1.7.32",
  "io.swagger.core.v3" % "swagger-core-jakarta" % "2.1.11",
  "com.fasterxml.jackson.module" %% "jackson-module-scala" % "2.13.0",
  "org.scalatest" %% "scalatest" % "3.2.10" % Test,
  "org.slf4j" % "slf4j-simple" % "1.7.32" % Test
)
```